src/Vlingo.Wire.Tests/Node/MockConfiguration.cs

```diff
@@ -0,0 +1,74 @@
+// Copyright © 2012-2018 Vaughn Vernon. All rights reserved.
+//
+// This Source Code Form is subject to the terms of the
+// Mozilla Public License, v. 2.0. If a copy of the MPL
+// was not distributed with this file, You can obtain
+// one at https://mozilla.org/MPL/2.0/.
+
+using System.Collections.Generic;
+using System.Linq;
+using Vlingo.Actors;
+
+namespace Vlingo.Wire.Tests.Node
+{
+    using Vlingo.Wire.Node;
+    
+    public class MockConfiguration : IConfiguration
+    {
+        private readonly ISet<Node> _nodes;
+
+        public MockConfiguration()
+        {
+            var node1 = Node.With(Id.Of(1), Name.Of("node1"), Host.Of("localhost"), 37371, 37372);
+            var node2 = Node.With(Id.Of(2), Name.Of("node2"), Host.Of("localhost"), 37373, 37374);
+            var node3 = Node.With(Id.Of(3), Name.Of("node3"), Host.Of("localhost"), 37375, 37376);
+            
+            _nodes = new SortedSet<Node>(new [] {node1, node2, node3});
+        }
+        
+        public IEnumerable<Node> AllNodesOf(IEnumerable<Id> ids)
+        {
+            return new List<Node>();
```

Comment:
Wouldn't change any behavior, but when you are revising, can we please change the one-liners to arrow notations?

Reply:
Yes, of course